python/ray/dashboard/agent.py

```diff
@@ -243,11 +244,15 @@ async def wait_forever():
             await self.http_server.cleanup()
 
 
-def open_capture_files(log_dir):
-    filename = f"agent-{args.agent_id}"
+def get_capture_filepaths(log_dir):
+    """Get filepaths for the given [log_dir].
+    log_dir:
+        Logging directory to place output and error logs.
+    """
+    filename = "dashboard-agent"
```

Comment:
This filename is not the old filename?

Reply:
I think you're right, the reason it doesn't surface is we're not testing out file:
https://github.com/ray-project/ray/blob/52d3e506a63769513e462278ad939e9bf781780a/python/ray/tests/test_logging.py#L145

Reply:
Another confusing point, we're not naming out file, log file and error file the same.

Example for runtime env agent:
```
runtime_env_agent.err
runtime_env_agent.log
runtime_env_agent.log.1
runtime_env_agent.log.2
runtime_env_agent.log.3
runtime_env_agent.out
runtime_env_agent.out.1
```
Example for dashboard agent:
```
agent-424238335.err
agent-424238335.out
dashboard_agent.log
dashboard_agent.log.1
dashboard_agent.log.2
dashboard_agent.log.3
```